app/src/main/java/com/codesoom/assignment/App.java

```diff
@@ -1,10 +1,31 @@
 package com.codesoom.assignment;
 
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import java.io.IOException;
+import java.net.InetSocketAddress;
+
 public class App {
     public String getGreeting() {
         return "Hello World!";
     }
 
     public static void main(String[] args) {
+
+        try {
+
+            InetSocketAddress serverAddress = new InetSocketAddress(8282);
+            HttpServer httpServer = HttpServer.create(serverAddress,0);
```

Comment:
```suggestion
            HttpServer httpServer = HttpServer.create(serverAddress, 0);
```

코드 포맷팅을 습관화 하시면 좋을 것 같습니다. 저는 저장될 때 마다 포맷팅 되게 해놨어요

Reply:
피드백 보고 바로 플러그인 설치했습니다 :) 유용한 거 알려주셔서 감사해요!